biocrnpyler/dna_construct.py

```diff
@@ -188,7 +188,7 @@ def get_species(self):
             partspec = copy.copy(part.dna_species)
             partspec.material_type = self.material_type
             ocomplx += [partspec.set_dir(part.direction)]
-        out_species = OrderedPolymerSpecies(ocomplx,base_species = self.base_species,circular = self.circular,\
+        out_species = OrderedPolymerSpecies(ocomplx,circular = self.circular,\
```

Comment:
sometimes you can name dna constructs and i want the species to have the same name. when it is bound to stuff then the name is auto generated

Reply:
ok - as long as we are sure the name gets auto-generated when bound.

Reply:
i guess naming it isnt that important because you can always say construct.get_species().name if you are trying to set its initial concentration or something. but then if the species isn't named, what is the point of a dna construct's name?

Reply:
for PolymerConformation, I think I got rid of the "name" attribute to enforce that automatic naming conventions are used. I think this is a fair thing to do with PolymerSpecies and DNAConstruct as well. I think name is probably a remnant of stuff normal Components use.

Reply:
ok i removed the name. i think everything is fine without it